### PR TITLE
More rework of new code

### PR DESCRIPTION
**Pull request by utopiabound**
Remove almost all pcs commands, as they don't seem to work in autotest.

This uses crm_resource to start/stop resources.
This uses cibadmin to add/remove resource and resource groups.

### CODE REVIEW

File: chroma_agent/action_plugins/manage_targets.py [hunk 1]
```diff
@@ -726,50 +764,39 @@ def start_target(ha_label):
...
 
-        # now wait for it to start
-        if _wait_target(ha_label, True):
-            location = get_resource_location(ha_label)
-            if not location:
-                return agent_error(
-                    "Started {} but now can't locate it!".format(ha_label)
-                )
-            return agent_result(location)
+                if i < 4:
```
Comment (jgrund) [inline chroma_agent/action_plugins/manage_targets.py]: Why 4?

Comment (utopiabound) [reply chroma_agent/action_plugins/manage_targets.py]: Comment says HYD-1898, I'll address the while loops in a separate patch.

File: chroma_agent/action_plugins/manage_targets.py [hunk 2]
```diff
@@ -430,19 +430,28 @@ def _configure_target_priority(primary, ha_label, node):
     return result
 
 
+def _nvpair_xml(element, key, value):
+    ET.SubElement(
+        element,
+        "nvpair",
+        {
+            "id": "{}-{}-{}".format(element.get("id"), element.tag, key),
```
[the remark this answers is not in the document]
Comment (utopiabound) [reply chroma_agent/action_plugins/manage_targets.py]: yes

**Comment by jgrund** [comment]
LGTM.

**Pull request merged**